index.js

```diff
@@ -3,6 +3,22 @@
 const program = require('commander');
 const fs = require('fs');
 const package = require('./package.json');
+const configName = './.pastshotsrc';
```

Comment:
No need to keep the constant outside of `readConfig`, as it is used only inside its context.